client/my-sites/upgrades/domain-management/email/add-google-apps-card.jsx

```diff
@@ -28,134 +28,174 @@ const AddGoogleAppsCard = React.createClass( {
 
 	render() {
 		const gapps = this.props.products.gapps,
-			googleAppsSupportUrl = support.ADDING_GOOGLE_APPS_TO_YOUR_SITE;
-		let price = gapps && gapps.cost_display;
+			googleAppsSupportUrl = support.ADDING_GOOGLE_APPS_TO_YOUR_SITE,
+			price = gapps && gapps.cost_display,
+			selectedDomainName = this.props.selectedSite.domain;
+
+		const displayPrice = price.replace( /(\d+[.,]?\d+)/, ( val ) => {
+			const number = parseFloat( val );
+			return number % 1 === 0 ? number : this.numberFormat( number, 2 );
+		} );
 
 		// Gapps price is stored annually but we'd like to show a monthly price
-		price = price.replace( /(\d+[.,]?\d+)/, ( val ) => {
+		const monthlyPrice = price.replace( /(\d+[.,]?\d+)/, ( val ) => {
 			const number = ( Math.round( parseFloat( val ) / 10 * 100 ) / 100 );
-			return number % 1 === 0 ? number : this.numberFormat( number, 2 );
+			return this.numberFormat( number, 2 );
 		} );
 
 		return (
-			<CompactCard className="add-google-apps-card">
-				<header className="add-google-apps-card__header">
-					<h3 className="add-google-apps-card__product-logo">
-						{ /* Intentionally not translated */ }
-						<strong>G Suite</strong>
-					</h3>
-
-					<div className="add-google-apps-card__price">
-						<h4 className="add-google-apps-card__price-per-user">
-							<span>
+			<div>
+				<CompactCard>
+					<header className="add-google-apps-card__header">
+						<h3 className="add-google-apps-card__product-logo">
+							{ /* Intentionally not translated */ }
+							<strong>G Suite</strong>
+						</h3>
+					</header>
+				</CompactCard>
+				<CompactCard>
+					<div className="add-google-apps-card__product-details">
+						<div className="add-google-apps-card__description">
+							<h2 className="add-google-apps-card__title">
+								{ this.translate( 'Professional email and so much more.' ) }
+							</h2>
+
+							<p>
 								{
 									this.translate(
-										'{{strong}}%(price)s{{/strong}} per user / month',
-										{
-											components: {
-												strong: <strong />
-											},
-											args: {
-												price: price
-											}
-										}
+										"We've partnered with Google to offer you email, " +
+										'storage, docs, calendars, and more integrated with your site.'
 									)
 								}
-							</span>
-						</h4>
+							</p>
 
-						<span className="add-google-apps-card__price-separator"> | </span>
-
-						<h5 className="add-google-apps-card__billing-period">
-							{ this.translate( 'Billed yearly — get 2 months free!' ) }
-						</h5>
-					</div>
-
-					{ this.renderAddGoogleAppsButton() }
-				</header>
+							<div className="add-google-apps-card__price">
+								<h4 className="add-google-apps-card__price-per-user">
+									<span>
+										{
+											this.translate(
+												'{{strong}}%(price)s{{/strong}} per month / user',
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( '{{strong}}%(price)s{{/strong}} per user / month' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).